tests/Endpoints/ObjectiveTest.php

```diff
@@ -41,9 +44,9 @@ public function putsToTest()
             'position' => ['position', $this->getFaker()->randomDigit],
             'notActive' => ['active', false],
             'competency' => ['competency', 1],
-            'courses' => ['courses', [3]],
-            'programYears' => ['programYears', [2]],
-            'sessions' => ['sessions', [2]],
+            'courseObjectives' => ['courseObjectives', [1], $skipped = true],
```

Comment:
Worried about skipping this as it's a pretty critical PUT path.

Reply:
good point, i've added an explicit test method for those.